.github/workflows/dax.yml

```diff
@@ -29,11 +41,21 @@ jobs:
     steps:
       - name: Check if the devdax exists, print out UMF_TESTS_DEVDAX_PATH and UMF_TESTS_DEVDAX_SIZE
         run: |
-          ndctl list -N --device-dax
+          echo UMF_TESTS_DEVDAX_NAMESPACE="${UMF_TESTS_DEVDAX_NAMESPACE}"
+          ndctl list --namespace=namespace${UMF_TESTS_DEVDAX_NAMESPACE} --device-dax
           ls -al /dev/dax${UMF_TESTS_DEVDAX_NAMESPACE}
           echo UMF_TESTS_DEVDAX_PATH="/dev/dax${UMF_TESTS_DEVDAX_NAMESPACE}"
           echo UMF_TESTS_DEVDAX_SIZE="$(ndctl list --namespace=namespace${UMF_TESTS_DEVDAX_NAMESPACE} | grep size | cut -d':' -f2 | cut -d',' -f1)"
 
+      - name: Check if the FSDAX exists
+        run: |
+          echo UMF_TESTS_FSDAX_NAMESPACE="${UMF_TESTS_FSDAX_NAMESPACE}"
+          echo UMF_TESTS_FSDAX_PATH="${UMF_TESTS_FSDAX_PATH}"
+          ndctl list --namespace=namespace${UMF_TESTS_FSDAX_NAMESPACE}
+          ls -al /dev/$(echo $UMF_TESTS_FSDAX_PATH | cut -d'/' -f3)
```

Comment:
there's a lot of greping here, perhaps you should make it the other way around - set `FSDAX_DEVICE=pmem1` and  use it to build all these vars here plus the full path just for testing below (`UMF_TESTS_FSDAX_PATH=/mnt/${device}/file`)

Reply:
Done